### ISSUE TITLE
Error: Status 403 trying to pull repository bigdatauniversity/spark: "Account is not Active"

### ISSUE THREAD

**Issue by rtjohn** [kitematic/kitematic#870]
Error: Status 403 trying to pull repository bigdatauniversity/spark: "Account is not Active"

**Comment by timfallmk** [comment]
@rtjohn Have you tried to log in with your account to Docker Hub, or are you attempting to pull as a guest?

**Comment by jeffdm** [comment]
+ Did you create a Docker Hub account via Kitematic?

**Comment by bwhitty** [comment]
I'm encountering the same thing. I created my Docker Hub account via Kitematic, and I get the same 403 error. Also, the password I set up via Kitematic does not work on hub.docker.com.  If I try to create an account on hub.docker.com with the same username / password, the form does say that the username and email are both already in use, but I never receive a password reset email if I request one. I can log in and out via Kitematic directly.
